src/t2iapi/context/service.proto

```diff
@@ -105,6 +105,26 @@ service ContextService {
   t2iapi.context.CreateContextStateWithAssocAndSpecificValidatorRequest)
       returns (t2iapi.context.CreateContextStateWithAssociationResponse);
 
+  /*
+  Manipulate a given context state (specified by state_handle), such that
+  - has the given context_association value,
+  - has the given number of pm:Validators (num_validators)
+  - its pm:Identification changes so that it differs from the last 2 pm:Identifications
+    of this particular context state.
+  Shall return the handle of the context state. This context state shall either be
```

Comment:
I think a few things have gotten mixed up here or i don't get it?! The manipulation specifies a state handle and returns one? But the rpc returns the BasicResponse.